graalvm/build-matrix/action.yml

```diff
@@ -13,7 +13,7 @@ runs:
         fetch-depth: 0
 
     - name: "Setup GraalVM CE"
-      uses: graalvm/setup-graalvm@v1.1.1
```

Comment:
This action is used just to find all modules that contain `nativeTest` gradle tasks and then to create a matrix. Maybe we should use some other java distribution here (instead of graalvm) to not make confusion. Other action is used to prepare environment for native build: https://github.com/micronaut-projects/github-actions/blob/9bfa798a275cc0ebb2b7b45c96834731e14d6945/graalvm/pre-build/action.yml#L34

Reply:
We could use this here
```
    - name: "Set up $JAVA_HOME for Gradle"
      uses: actions/setup-java@v3.12.0
      with:
        distribution: 'oracle'
        java-version: '17'
```